package.json

```diff
@@ -59,6 +61,8 @@
     "@babel/preset-typescript": "7.12.7",
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
+    "@cypress/react": "^5.9.2",
+    "@cypress/webpack-dev-server": "^1.4.0",
```

Comment:
Is this used?

Reply:
yes, it is used :) 
1. import { mount } from "@cypress/react"; for `mount()`'ing;
2. Webpack is used for proper component testing